.pre-commit-config.yaml

```diff
@@ -34,7 +34,7 @@ repos:
   rev: v2.17.0
   hooks:
     - id: pylint
-      args: [--disable=import-error]
+      args: [--disable=import-error, --disable=R0801]
```

Comment:
I cannot accept any changes in this file in this PR. Could you please revert this?